Mathlib/MeasureTheory/Integral/Asymptotics.lean

```diff
@@ -0,0 +1,65 @@
+/-
+Copyright (c) 2024 Lawrence Wu. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Lawrence Wu
+-/
+import Mathlib.MeasureTheory.Integral.IntegrableOn
+
+/-!
+# Bounding of integrals by asymptotics
+
+We establish integrability of `f` from `f = O(g)`.
+
+## Main results
+
+* `Asymptotics.IsBigO.integrableAtFilter`: Given measurable functions `f`, `g` with `f = O[l] g`,
+  `IntegrableAtFilter g l μ` implies `IntegrableAtFilter f l μ`.
+-/
+
+open Asymptotics MeasureTheory Set Filter
+
+variable {α E : Type*} [MeasurableSpace α] [NormedDivisionRing E] [NormedSpace ℝ E]
+ {f g : α → E} {a b : α} {μ : Measure α} {l : Filter α}
```

Comment:
Why do you need `E` to be a normed real algebra, not a normed space? Also, `f` and `g` can have different codomain.

Reply:
Thanks; I've managed to reduce it to `[NormedAddCommGroup E] [NontriviallyNormedField F]`

Reply:
Should work for an `NormedAddCommGroup F`, if you add `IntegrableAtFilter.norm` and use it here.

Reply:
Right, of course 🤦‍♂️ ; fixed, thanks!